react/components/Payment/ConnectorResponseInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React, { FunctionComponent } from 'react'
+import { injectIntl, InjectedIntlProps, intlShape } from 'react-intl'
+
+const ConnectorResponseInfo: FunctionComponent<Props & InjectedIntlProps> = ({
+  label,
+  value,
+  intl,
+}) => {
+  return (
+    <div className="more-info__data__item f7">
+      <span className="more-info__data__item__label mr2 fw5">
+        {intl.formatMessage({
```

Comment:
Use the FormattedMessage component instead and doing so you can remove the inject intl.